Add first round of guide and remove subtree docs directory

maxwolfs opened the pull request:
closes #2

ra-beer commented on docusaurus.config.js:
```diff
@@ -37,92 +37,94 @@ const config = {
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/SovereignCloudStack/Docs/tree/main/',
+                        'https://github.com/SovereignCloudStack/Docs/tree/main/'
```
I think the indentation is a bit broken here

ra-beer commented on docusaurus.config.js:
```diff
@@ -37,92 +37,94 @@ const config = {
           // Remove this to remove the "edit this page" links.
           editUrl:
-            'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+                        'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/'
```
Same here

ra-beer reviewed (commented):
I read the code and as far as I can say it looks good. Let's test if its works. Only the two indentations what I found seems to be incorrect.

maxwolfs commented:
@ra-beer: I have corrected the false formatting. I will open up an issue for the general linting and formatting of .js and .ts files, as it is not working properly yet.

@aeroflydesign: I have also added the installation guide within this PR now

ra-beer reviewed (approved):
LGTM Thanks!
